tests/unit/amo/components/TestAddonMoreInfo.js

```diff
@@ -315,6 +317,34 @@ describe(__filename, () => {
     expect(statsLink).toHaveProp('href', '/addon/coolio/statistics/');
   });
 
+  it('links to stats if user has STATS_VIEW permission', () => {
+    const addon = createInternalAddon({
+      ...fakeAddon,
+      public_stats: false,
+    });
+    const root = render({
+      addon,
+      store: dispatchSignInActions({ permissions: [STATS_VIEW] }).store,
+    });
+
+    const statsLink = root.find('.AddonMoreInfo-stats-link');
+    expect(statsLink).toHaveLength(1);
+  });
+
+  it('links to stats if user is ADMIN', () => {
```

Comment:
This test should be deleted. It would be testing an implementation detail of `hasPermission()` which is not necessary for this unit test. More importantly, the assertion is a duplication of https://github.com/mozilla/addons-frontend/pull/3741/files#diff-2e12f8c050405872bcaf7b56035e9387R157